test/integration/targets/connection_ssh/fix_logingracetime.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: all
+  tasks:
+    - lineinfile:
+        path: /etc/ssh/sshd_config
+        regexp: '^LoginGraceTime'
+        line: LoginGraceTime 2m
+
+    - when: ansible_facts.system == 'Darwin'
+      block:
+        - command: launchctl stop com.openssh.sshd
+        - command: launchctl start com.openssh.sshd
+
+    - service:
+        name: ssh{{ '' if ansible_facts.os_family == 'Debian' else 'd' }}
+        state: restarted
+      when: ansible_facts.system != 'Darwin'
```

Comment:
This certainly shouldn't be addressed here. I'll need to find the appropriate location to do this, and not have it be a side effect of a test.

This is addressing the Fedora41 CI failures for:

```
Jan 20 15:45:55 75e3a58df295 sshd[23]: drop connection #0 from [127.0.0.1]:45472 on [127.0.1.1]:22 penalty: exceeded LoginGraceTime
```

I have no clue why this PR triggers this, and why explicitly setting this to the default value is somehow different than it just not being provided.